client/my-sites/customer-home/cards/tasks/titan-banner/index.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+// import config from '@automattic/calypso-config';
+import Task from 'calypso/my-sites/customer-home/cards/tasks/task';
+import { TASK_UPSELL_TITAN } from 'calypso/my-sites/customer-home/cards/constants';
+import titanUpsellIllustration from 'calypso/assets/images/customer-home/illustration--titan-banner.svg';
+
+const TitanBanner = () => {
+	const translate = useTranslate();
+	/* Not sure if there  needs to be any such condition or based on a feature flag. */
+	const showTitanBanner = true;
+
+	return (
+		<>
+			{ showTitanBanner && (
+				<Task
+					title={ translate( 'Get email @ your domain' ) }
+					description={ translate(
+						'Brand yourself and build trust with a custom email address @ your domain, free for 3 months.'
+					) }
+					actionText={ translate( 'Get email' ) }
```

Comment:
My understanding is the copy is still being wrangled, so we can defer tweaks on that for now.

Reply:
I would still love to understand the translation bits as well, might come in handy further down the line.